slothy/targets/arm_v7m/arch_v7m.py

```diff
@@ -0,0 +1,1070 @@
+import logging
+import inspect
+import re
+import math
+from enum import Enum
+from functools import cache
+
+from sympy import simplify
+
+llvm_mca_arch = "arm"  # TODO
+
+
+class RegisterType(Enum):
+    GPR = 1
+    FPR = 2
+    STACK_FPR = 3
```

Comment:
Are those stack and hint registers still needed with the advent of `// @slothy: reads/writes=....` annotations?

Reply:
I think so, there are probably some more artifacts in the Arch model that stem from basing it on the aarch64 one. 
I am going to fix this when I continue work on the Arch model.